apps/web/components/mail/MailDetail.tsx

```
@@ -39,38 +139,105 @@ export const MailDetail = () => {
+                    </div>
+                  </div>
+                  <AnimatePresence initial={false}>
+                    {openMessageIndex === index && (
+                      <motion.div
+                        key="content"
+                        initial={{ opacity: 0, height: 0 }}
+                        animate={{ opacity: 1, height: "auto" }}
+                        exit={{ opacity: 0, height: 0 }}
+                        transition={{ duration: 0.2 }}
+                        className="text-sm text-gray-200 leading-relaxed px-2 mt-3 overflow-hidden"
+                        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)